kinto/__main__.py

```diff
@@ -0,0 +1,49 @@
+import argparse
+import sys
+import subprocess
+
+from pyramid.paster import bootstrap
+
+
+def kinto_migrate(env):
```

Comment:
I may be wrong, but I think you could use the `cliquet_migrate` function here (in `cliquet/scripts`)